libnavui-androidauto/src/main/java/com/mapbox/androidauto/preview/CarRoutePreviewScreen.kt

```diff
@@ -86,21 +86,13 @@ internal class CarRoutePreviewScreen @UiThread constructor(
         navigationRoutes.forEach { navigationRoute ->
             val route = navigationRoute.directionsRoute
             val title = route.legs()?.first()?.summary() ?: placeRecord.name
-            val duration = CarDistanceFormatter.formatDistance(route.duration())
-            val routeSpannableString = SpannableString("$duration $title")
-            routeSpannableString.setSpan(
-                DurationSpan.create(route.duration().toLong()),
-                0,
-                duration.length,
-                0
-            )
+            val routeSpannableString = SpannableString("  $title")
+            val span = DurationSpan.create(route.duration().toLong())
+            routeSpannableString.setSpan(span, 0, 1, 0)
```

Comment:
Will you add some screen shots on what the issue was and how this fixes it? Specifically add it to the description of the PR though. We'll probably try to change this `routeSpannableString` again

https://github.com/mapbox/1tap-android/pull/1305
<img width="827" alt="Screenshot 2023-03-15 at 3 36 46 PM" src="https://user-images.githubusercontent.com/3021882/225458870-60a08e75-0136-4ded-9631-1c503a0cc4a1.png">

Reply:
The issue was that we used `CarDistanceFormatter.formatDistance(route.duration())` for `Row.Builder.addText` instead of `CarDistanceFormatter.formatDistance(route.distance())`. There is no difference when expected speed is 1 m/s (3.6 km/h or 2.2 mi/h). Added screenshots.

Reply:
Oh `duration` vs `distance` 👍 

Nice